### DISCUSSION TITLE
UNABLE to verify your academic affiliation

### DISCUSSION THREAD

**Discussion by googidaddy** [community/community#52301] [GitHub Education]
- Attendees of this school must verify and use their academic email address.

- Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

I don't have a academic email address and I submit a Ministry of Education online verification report. This file have my school all info.

**Comment by yelbom** [comment]
Hi everyone my name is Scott and I feel like the internet has been after me for 8 years. Is this true?

**Comment by samus-aran** [comment]
Hi @googidaddy 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍
